tests/CMakeLists.txt

```diff
@@ -121,5 +123,8 @@ set(enabled_tests
     fuzz
     unit_tests)
 
+=======
+>>>>>>> 060c3f7... squash
```

Comment:
That's a merge conflict. This whole file can be removed from the patch.

Reply:
ugh lazy. my bad.